models/user.js

```diff
@@ -6,6 +6,12 @@ const userSchema = new Schema({
   },
   password: {
     type: String
+  },
+  isMailVerified: {
```

Comment:
don't change the user schema.
all users all verified by default.
we will handle this in the future.